cmd/archive_test.go

```diff
@@ -40,10 +41,17 @@ func TestArchiveThresholds(t *testing.T) {
 		t.Run(testCase.name, func(t *testing.T) {
 			t.Parallel()
 
+			tmpDir, err := os.MkdirTemp("", "k6archivetest")
+			require.NoError(t, err)
+			defer func() {
+				require.NoError(t, os.RemoveAll(tmpDir))
+			}()
+			tmpPath := filepath.Join(tmpDir, "archive.tar")
+
```

Comment:
maybe use [t.TempDir](https://pkg.go.dev/testing#T.TempDir)?

Reply:
Join @MStoykov here `t.TempDir` seems cleaner

Reply:
:+1: very good point, forgot that existed :sweat_smile: Fixed by the [latest force push](https://github.com/grafana/k6/compare/42ff1bd3e62408450d7f61fb7df9eaca7aa023de..c1e305197a49aa38472db3fe8c4ba6289d9c852d)